### DISCUSSION TITLE
how to make openrgb devices be recognized?

### DISCUSSION THREAD

**Discussion by wesioplayconstructor** [ChromaControl/ChromaControl#266] [Q&A]
I wanted to use openrgb to control esps32 that controls the argb led strip with the wled program, because the program to integrate wled to the razer synapse is not good yet, and my redragon mouse has the redragon cobra that has predefined effects but to control static effects, would there be any way?

**Comment by andrewbabbittdev** [comment]
OpenRGB support has been in progress for the past few weeks, still more work to be done but posting frequent updates in the discord server.
